specification/validated-relationships-service-api.yaml

```diff
@@ -296,6 +297,7 @@ components:
         authored: 
           type: string
           description: The date and/or time that this set of answers were last changed.
+          example: "2024-07-15T09:43:03.280Z"
           # On Confluence the example has this as a string but here:
```

Comment:
Think this comment be removed now can't it? `source` is an object and what was previously in confluence was very rough rather than prescriptive

Reply:
@miiisterjim so what `type` should `authored` be? Should it be a `dateTime` as per:

https://simplifier.net/HL7FHIRUKCoreR4/UKCore-QuestionnaireResponse/~details

Reply:
@chris-young-12-nhs good spot, yes it should I reckon

Reply:
As per the [OAS spec](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#data-types), date-times have to be strings - but *can* have a format of date-time - so this is what I'll implement:

```
        authored: 
          type: string
          format: date-time
          description: The date time that this set of answers were last changed.
          example: "2024-07-15T09:43:03.280Z"
```